modules/i3s/src/i3s-loader.js

```diff
@@ -90,3 +93,33 @@ async function parseTile(data, options, context) {
   data = JSON.parse(new TextDecoder().decode(data));
   return normalizeTileData(data, options, context);
 }
+
+async function parseFeatureAttributes(tile, tileset) {
+  const attributeStorageInfo = tileset.attributeStorageInfo;
```

Comment:
Idea: We could an option to the I3SLoader to control loading of these attributes to speed things up when not needed?